Sources/OpenCombine/CombineIdentifier.swift

```diff
@@ -9,6 +9,14 @@
 import COpenCombineHelpers
 #endif
 
+#if os(WASI)
+private var __identifier: UInt64 = 0
+func __nextCombineIdentifier() -> UInt64 {
+    __identifier += 1
+    return __identifier
```

Comment:
For consistency with another implementation of this function:
```suggestion
    defer { __identifier += 1 }
    return __identifier
```